src/spec/dsl.cr

```diff
@@ -64,6 +65,28 @@ module Spec
     exit
   end
 
+  # :nodoc:
+  class_getter randomizer_seed : UInt64?
+  class_getter randomizer : Random::PCG32?
+
+  # :nodoc:
+  def self.order=(mode)
+    seed =
+      case mode
+      when "default"
+        nil
+      when "random"
+        Random::Secure.rand(1..99999).to_u64 # 5 digits or less for simplicity
+      when UInt64
```

Comment:
I'd use `Int`

Reply:
I opened up the number based on the feedback from @straight-shoota :

> Also I don't think there needs to be a limit enforced to the allowed seed number.

I chose UInt64 for 2 reasons. 1) The unsigned part prevents negative numbers and 2) the input to Randon::PCG32 takes a UInt64, so it's cleaner to convert once (as opposed to Int, then to UInt64 later)